README.md

```diff
@@ -1 +1 @@
-# aMAZEthing
\ No newline at end of file
+# aMAZEthing
```

Comment:
Should we get this done now or wait for the final week?

Reply:
Add to taks